db/models/apartment.rb

```diff
@@ -0,0 +1,3 @@
+class Apartment
```

Comment:
Oh, and we usually keep our `models` folder separate (outside of) our `db` folder.